### PR TITLE
feat:anilist tracking service

### PR DESCRIPTION
**Pull request by appdevelpo**
screen shot:

contain lots of bug 
only work on mobile now

### CODE REVIEW

**Comment by appdevelpo** [comment]
screen shot for desktop ver
image
image
image

**Comment by OshekharO** [comment]
Screenshot_20240108-003509_Miru

I saw these two errors after authorizing, also clicking on that anilist thing at bottom right doesn't do anything.

**Comment by OshekharO** [comment]
Screenshot_20240108-003739_Miru

Also this alignment broken

**Comment by appdevelpo** [comment]
image

**Comment by OshekharO** [comment]
Login not working.

File: lib/views/pages/settings/settings_page.dart [hunk 1]
```diff
@@ -358,6 +358,17 @@ class _SettingsPageState extends State<SettingsPage> {
         androidIcon: Icons.sync,
         content: Column(
           children: [
+            SettingsSwitchTile(
+              title: 'tracker-setting.auto-update-progress'.i18n,
```
Comment (bachig26) [inline lib/views/pages/settings/settings_page.dart:362]: @appdevelpo 

k54wRxu7IA

**Review by MiaoMint — APPROVED** [review]

**Pull request merged**